drivers/overlay/ov_network.go

```diff
@@ -470,7 +477,7 @@ func (n *network) generateVxlanName(s *subnet) string {
 		id = n.id[:5]
 	}
 
-	return "vx-" + fmt.Sprintf("%06x", n.vxlanID(s)) + "-" + id
+	return "vx-" + fmt.Sprintf("%06x", s.vni) + "-" + id
```

Comment:
considering that we are touching this line, this will be more efficient:
`fmt.Sprintf("vx-%06x-%v", s.vni,id)`